packages/SwingSet/test/test-kernel.js

```diff
@@ -1064,20 +1097,23 @@ test('pipelined promise queueing', async t => {
     {
       id: p1ForKernel,
       state: 'unresolved',
+      refCount: '2',
```

Comment:
Does this `refCount` going from 3 to 2 effectively test the refcount being decremented after some other promise gets retired? I see tests getting updated to match the new expected state contents, but I want to make sure we have at least one test that explicitly exercises the intended behavior (instead of merely tolerating it). Let's make sure at least one of these tests checks the refcount before and after retirement.